src/noisepy/seis/__init__.py

```diff
@@ -0,0 +1,9 @@
+# no-qa F401
+from .S0A_download_ASDF_MPI import download  # noqa: F401
+from .S1_fft_cc_MPI import cross_correlate  # noqa: F401
+from .S2_stacking import stack  # noqa: F401
+
+__version__ = "0.5.4"
```

Comment:
This manual package versioning is probably fine for now, but eventually it would be better to have auto versioning based on the version control system. This way, when you make a release it will just grab the version from the release tag. Now that you're using hatch, I suggest adding `hatch-vcs` https://github.com/ofek/hatch-vcs

Reply:
Gotcha, I'll do that in a subsequent PR